static/app/components/errorRobot.tsx

```diff
@@ -146,7 +146,6 @@ const ErrorRobotWrapper = styled('div')<{gradient: boolean}>`
   display: flex;
   justify-content: center;
   font-size: ${p => p.theme.fontSizeExtraLarge};
-  box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.08);
```

Comment:
<img width="686" alt="Screen Shot 2021-09-28 at 4 00 21 PM" src="https://user-images.githubusercontent.com/44172267/135176949-c0d1babf-6cf8-477d-8f38-f17aaf9ce4ea.png">

(see: the slight inset shadows around the outer borders)